app/presenters/content_item/news_image.rb

```diff
@@ -7,17 +7,32 @@ def image
   private
 
     def default_news_image
-      organisation = content_item.dig("links", "primary_publishing_organisation")
-      organisation[0].dig("details", "default_news_image") if organisation.present?
+      return first_worldwide_organisation_default_news_image if world_news_story?
+
+      first_primary_publishing_organisation_default_news_image
     end
 
     def placeholder_image
       # this image has been uploaded to asset-manager
-      if content_item["document_type"] == "world_news_story"
+      if world_news_story?
         { "url" => "https://assets.publishing.service.gov.uk/media/5e985599d3bf7f3fc943bbd8/UK_government_logo.jpg" }
       else
         { "url" => "https://assets.publishing.service.gov.uk/media/5e59279b86650c53b2cefbfe/placeholder.jpg" }
       end
     end
+
+    def first_worldwide_organisation_default_news_image
+      worldwide_organisation = content_item.dig("links", "worldwide_organisations")
+      worldwide_organisation[0].dig("details", "default_news_image") if worldwide_organisation.present?
```

Comment:
Minor nitpick where I think `first` would be nicer than accessing with a numeric position.  The `first` method still returns `nil` if there's an empty array.
```suggestion
      worldwide_organisation = content_item.dig("links", "worldwide_organisations").first
      worldwide_organisation.dig("details", "default_news_image") if worldwide_organisation.present?
```